apps/cookbook/src/app/showcase/link-showcase/link-showcase.component.html

```diff
@@ -54,3 +54,20 @@ <h2>New tab</h2>
   In a non-web context, i.e. an app, any link will most likely launch the device's browser and
   effectively open a new tab outside the app context.
 </p>
+
+<h2>Custom trailing-icon</h2>
+<p>
+  If you want to use a custom icon at the end instead of the link then extend the mixin class
```

Comment:
Consider readability:
```suggestion
  If you want to use a custom icon after the link instead of the external link icon then include the Kirby scss @mixin <code>utils.trailing-icon</code>  
```